dll/appcompat/apphelp/shimeng.c

```diff
@@ -1175,7 +1181,7 @@ VOID SeiResetEntryProcessed(PPEB Peb)
     }
 }
 
-VOID SeiInit(PUNICODE_STRING ProcessImage, HSDB hsdb, SDBQUERYRESULT* pQuery)
+VOID SeiInit(LPCWSTR ProcessImage, HSDB hsdb, SDBQUERYRESULT* pQuery, BOOLEAN ProcessInit)
```

Comment:
Looks like you could have kept `ProcessImage` as pointer to a `UNICODE_STRING`, and in `SE_DynamicShim` initialize a ``UNICODE_STRING` local variable to give to `SeiInit`. This would also allow `SE_InstallBeforeInit` to just keep giving the unicode string as parameter.